packages/components/src/components/DistributionChart.tsx

```diff
@@ -44,15 +45,62 @@ export const DistributionChart: React.FC<DistributionChartProps> = ({
   showControls,
   logX,
   expY,
+  truncateToNthci,
 }) => {
   const [isLogX, setLogX] = React.useState(logX);
   const [isExpY, setExpY] = React.useState(expY);
 
   React.useEffect(() => setLogX(logX), [logX]);
   React.useEffect(() => setExpY(expY), [expY]);
 
-  const shape = distribution.pointSet();
   const [sized] = useSize((size) => {
+    const delta = truncateToNthci / 100 / 2;
+    if (truncateToNthci <= 0 || truncateToNthci > 100) {
+      return (
+        <ErrorAlert heading="Confidence interval error">
```

Comment:
Nice use of lots of error messages. That said, I think a lot of this functionality should probably be abstracted into a separate function.